cmd/rancher-desktop-guestagent/main.go

```diff
@@ -75,29 +80,60 @@ func main() {
 
 	group, ctx := errgroup.WithContext(context.Background())
 
-	if *enableDocker {
+	if *enablePriviledService && *enableIptables {
+		log.Fatal("error enabling Privileged Service mode and iptables, you can only enable one.")
```

Comment:
```suggestion
		log.Fatal("-privilegedService and -iptables are mutually exclusive; you can only enable one.")
```
The last clause is optional; _mutually exclusive_ by itself should be sufficient.